test/clj/cider/nrepl/middleware/util/namespace_test.clj

```diff
@@ -5,3 +5,10 @@
 (deftest test-project-namespaces
   (is (contains? (into #{} (n/project-namespaces))
                  'cider.nrepl.middleware.util.namespace)))
+
+(deftest test-loaded-namespaces
+  ;; If we don't pass the second arg, some cider ns will be returned
+  (is (some #(re-find #".*nrepl" %) (n/loaded-namespaces)))
+  ;; Shouldn't return any cider.nrepl namespaces
+  (is (not-any? #(re-find #".*nrepl" %)
```

Comment:
@Malabarba Changed it. I got an email about your comment. But cannot actually see it on GH. Weird.